seacrowd/sea_datasets/bud500/bud500.py

```diff
@@ -0,0 +1,174 @@
+                TASK_TO_SCHEMA[_SUPPORTED_TASKS[0]]
+            ]  # speech_text_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        file_list = HfFileSystem().ls("datasets/linhtran92/viet_bud500/data", detail=False)
+        train_urls, test_urls, val_urls = [], [], []
+
+        for filename in file_list:
+            if filename.endswith(".parquet"):
+                filename = filename.split("/")[-1]
+                split = filename.split("-")[0]
+                url = _BASE_URL.format(filename=filename)
+
+                if split == "train":
+                    train_urls.append(url)
+                elif split == "test":
+                    test_urls.append(url)
+                elif split == "validation":
+                    val_urls.append(url)
+
+        train_paths = list(map(Path, dl_manager.download(sorted(train_urls))))
+        test_paths = list(map(Path, dl_manager.download(sorted(test_urls))))
+        val_paths = list(map(Path, dl_manager.download(sorted(val_urls))))
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={"data_paths": train_paths},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={"data_paths": test_paths},
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={"data_paths": val_paths},
+            ),
+        ]
+
+    def _generate_examples(self, data_paths: Path) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+        key = 0
+        for data_path in data_paths:
+            with open(data_path, "rb") as f:
+                pf = pq.ParquetFile(f)
+
+                for row_group in range(pf.num_row_groups):
+                    df = pf.read_row_group(row_group).to_pandas()
+
+                    for row in df.itertuples():
+                        if self.config.schema == "source":
+                            yield key, {
+                                "audio": row.audio,
+                                "transcription": row.transcription,
+                            }
+                        elif self.config.schema == _SEACROWD_SCHEMA:
+                            yield key, {
+                                "id": str(key),
+                                # "path": None,
+                                "audio": row.audio,
+                                "text": row.transcription,
+                                # "speaker_id": None,
+                                # "metadata": None,
```

Comment:
Please uncomment these variables. These comments cause the unit tests to fail.
For all else, LGTM! Thanks @akhdanfadh!
P.S. Downloading the original 100+ files probably took so long for you too, kudos for making this work!